service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/IotHubConnectionStringBuilder.java

```diff
@@ -26,12 +26,10 @@
      *
      * @param connectionString The connection string
      * @return The IotHubConnectionString object
-     * @throws IOException This exception is never thrown. Users are advised to use {@link #createIotHubConnectionString(String, AuthenticationMethod)} instead
-     * since it's method signature does not declare any thrown exceptions.
      * @deprecated Use {@link #createIotHubConnectionString(String)} since it does not declare any thrown exceptions
      */
     @Deprecated
-    public static IotHubConnectionString createConnectionString(String connectionString) throws IOException
```

Comment:
This class is about 1 layer removed from being public, so I'd revert this change. We reference it in a sample or two when demonstrating how to build SAS tokens